web/sites/default/config/core.entity_view_display.media.document.teaser.yml

```diff
@@ -39,20 +44,56 @@ content:
     settings:
       use_description_as_link_text: true
     third_party_settings: {  }
-  field_summary:
+  field_document_type:
+    type: entity_reference_label
     weight: 1
+    region: content
     label: hidden
     settings:
-      link_to_entity: false
-    third_party_settings: {  }
+      link: false
+    third_party_settings:
+      ds:
```

Comment:
Aren't these settings better off in a theme template instead of display suite?

Reply:
Actually, our templates are out of control, so I would rather we had some of the basic layout in ds fields than a 200 line twig template with lots of if statements :)

That said, we'll likely refactor a lot that out into field level templates going forward.